osu.Game/Overlays/SettingsPanel.cs

```diff
@@ -34,6 +34,17 @@ public abstract class SettingsPanel : OsuFocusedOverlayContainer
 
         protected override Container<Drawable> Content => ContentContainer;
 
+        /// <summary>
+        /// The <see cref="SettingsPanel"/> always needs to be present for <see cref="ContentContainer"/> to process transforms while overlay is masked away.
+        /// todo: there may be a better solution for this and the existing <see cref="NonMaskedContent"/>, likely requires a refactor.
+        /// </summary>
```

Comment:
This has always been the issue for `SettingsPanel`, but it starts to matter now as it makes testing the screen offset current X position impossible, due to some of the move transforms getting hanged and not finishing, leading to non-zero/non-final position values.

Might require a refactor for this to not rely on overriding `IsPresent` to always true or not making `ContentContainer` masked whatsoever, but leaving that for later as it's pretty out of scope and I don't want to touch it further here.